routes/index.js

```diff
@@ -19,6 +19,19 @@ router.get('/rules', function(req, res) {
 });
 
 router.post('/invite', function(req, res) {
+
+  var checkEmail = require('legit');
+  // Checks for the existence of a MX record for the email address' domain
+  checkEmail(req.body.email, function(validation, addresses, err) {
+    if (!validation) {
+      res.render('result', {
+        community: config.community,
+        message: 'Failed. Email domain is invalid.',
+        isFailed: true
+      });
+    }
```

Comment:
@kexline4710 it worked locally for me but ... NODE. When I have recovered from my NODE anguish I'll triple check it. AKA I'll confirm this tomorrow.